### PR TITLE
Cross-selling: Use different label if there are no add-on products

### PR DESCRIPTION
**Pull request by raphaelm**
Before:

image

After:

Screenshot 2024-10-28 at 12-13-22 Step 1 of 5 Add-on products – Checkout Cross-selling test shop

### CODE REVIEW

File: src/pretix/presale/templates/pretixpresale/event/checkout_addons.html [hunk 1]
```diff
@@ -16,7 +16,7 @@
             {% trans "A product in your cart is only sold in combination with add-on products that are no longer available. Please contact the event organizer." %}
         </div>
     {% endif %}
-    <form class="form-horizontal" method="post" data-asynctask
+    <div class="form-horizontal" method="post" data-asynctask
```
Comment (wiffbi) [inline src/pretix/presale/templates/pretixpresale/event/checkout_addons.html:19]: Why did you remove the form? It does not work without it in my tests.

```suggestion
    <form class="form-horizontal" method="post" data-asynctask
```

**Review by wiffbi — CHANGES_REQUESTED** [review]
I guess it was accidental, that you remoced the form-element? Otherwise the addon-step does not work in my tests.

**Comment by raphaelm** [comment]
argh, when the IDE tries to be clever but isn't...

File: src/pretix/presale/templates/pretixpresale/event/checkout_addons.html [hunk 2]
```diff
@@ -57,16 +57,28 @@ <h3 class="panel-title">
         </div>
 
         {% if cross_selling_data %}
-            <details class="panel panel-default cross-selling" open>
+            {% if forms %}
+                <details class="panel panel-default cross-selling" open>
+            {% else %}
+                <div class="panel panel-default cross-selling">
```
Comment (wiffbi) [inline src/pretix/presale/templates/pretixpresale/event/checkout_addons.html:63]: Hm, just stumbled upon this. It creates invalid HTML in combination with the summary element. Not that it’s super bad, but not ideal for a11y. So could you move the whole summary-block with headline „more recomandations“ into the if-block and in the else-block replace the summary-tag with a div.

**Review by wiffbi — CHANGES_REQUESTED** [review]
Looks good, but just stumbled upon a potential a11y problem regarding details/summary.

**Review by wiffbi — APPROVED** [review]

**Pull request merged**